recipes/hdf5-plugins/meta.yaml

```diff
@@ -0,0 +1,91 @@
+{% set name = "hdf5-plugins" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/nexusformat/HDF5-External-Filter-Plugins/archive/v{{ version }}.tar.gz
+  sha256: 3d0d93cb71e1f384e827fbc9808d72b0984195ab337cca697c66205e7c135b0d
+
+build:
+  skip: True  # [win]
+  number: 0
+
+requirements:
+  run:
+    - {{ pin_subpackage(name|lower + '-bitshuffle', exact=True) }}
+    - {{ pin_subpackage(name|lower + '-bzip2', exact=True) }}
+    - {{ pin_subpackage(name|lower + '-lz4', exact=True) }}
+
+outputs:
+  - name: {{ name|lower + '-bitshuffle' }}
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+        - cmake >=3.0.0
+      host:
+        - hdf5
+        - lz4-c
+      run:
+        - hdf5
+        - lz4-c
+    script: build-bitshuffle.sh   # [not win]
+    test:
+      commands:
+        - conda inspect linkages -p $PREFIX {{ name|lower + '-bitshuffle' }}
+
+  - name: {{ name|lower + '-bzip2' }}
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+        - cmake >=3.0.0
+      host:
+        - bzip2
+        - hdf5
+      run:
+        - bzip2
+    script: build-bzip2.sh   # [not win]
+    test:
+      commands:
+        - conda inspect linkages -p $PREFIX {{ name|lower + '-bzip2' }}
+
+  - name: {{ name|lower + '-lz4' }}
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+        - cmake >=3.0.0
+      host:
+        - hdf5
+        - lz4-c
+      run:
+        - lz4-c
+    script: build-lz4.sh   # [not win]
+    test:
+      commands:
+        - conda inspect linkages -p $PREFIX {{ name|lower + '-lz4' }}
+
+about:
+  home: https://github.com/nexusformat/HDF5-External-Filter-Plugins
+  license: various
```

Comment:
I think you can put a list of licenses here

Reply:
couldn't that be misunderstood as multi-licensing? ie. I would want to express "some parts of this are BSD and others are GPL" as opposed to "BSD, GPL dual-licensed".

In addition to that I don't feel qualified to even translate the collection of licenses (eg. https://github.com/nexusformat/HDF5-External-Filter-Plugins/blob/master/LZ4/Additional_Legal/h5py_Copyrights_and_Licenses.txt vs the main license file at https://github.com/nexusformat/HDF5-External-Filter-Plugins/blob/master/resources/LICENSE.txt) to their respective license types